dijkstra/dijkstra.c

```diff
@@ -16,7 +16,7 @@
 int minimum_distance(const int min_distances[], const bool shortest_paths[]) {
 	int i;
 	int minimum = INFINITY, index;
-	for (int i = 0; i < VERTICES; ++i) {
+	for (i = 0; i < VERTICES; ++i) {
```

Comment:
@singhpratyush I checked it twice sir, don't know how it is showing it here? I even tested on various inputs. 😕